plumber-android-startup/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          xmlns:tools="http://schemas.android.com/tools"
+          package="com.squareup.leakcanary.plumber">
+    <application>
+        <provider
+                android:name="leakcanary.internal.PlumberInstaller"
+                android:authorities="${applicationId}.plumber-installer"
+                tools:node="remove"
+                android:exported="false"/>
```

Comment:
Maybe instead we should split Plumber into a core (where most of the code lives) and `:plumber-android` which adds the provider and enables it in the manifest. That way there's no noise, no provider at all in the startup version.

Reply:
(same for app watcher btw)